Rakefile

```diff
@@ -65,6 +65,9 @@ task 'setup_readline_and_openssl' do
     end
 
     unless Dir.exist?("./#{$OPENSSL_PACKAGE_NAME}")
+      unless File.exist?("#{$OPENSSL_PACKAGE_NAME}.tar.gz")
+        system('wget https://www.openssl.org/source/openssl-1.0.2l.tar.gz')
```

Comment:
OSX doesn't have `wget` command as a default. `curl` would be better.

```console
$ which -a wget
/usr/local/bin/wget
$ which -a curl
/usr/bin/curl
```

Reply:
Or they are still possible solution.

- do it by pure Ruby
- bundle openssl-1.0.2l.tar.gz same as 1.0.2d